src/service/ie-media-bug.js

```diff
@@ -29,6 +29,10 @@ const TAG = 'ie-media-bug';
  * @package
  */
 export function ieMediaCheckAndFix(win, opt_platform) {
```

Comment:
I believe this one is also handled a few lines below the intrinsic check in resources-impl.

https://github.com/ampproject/amphtml/blob/eca5174f4c41ef00ceb9ec4a18b3282aec608fc8/src/service/resources-impl.js#L362-L370